plugins/blueprint.cpp

```diff
@@ -1107,78 +1104,48 @@ static const char * get_tile_place(const df::coord &pos,
     return add_expansion_syntax(ctx, get_place_keys(ctx));
 }
 
-/* TODO: understand how this changes for v50
-static bool hospital_maximums_eq(const df::hospital_supplies &a,
-                                 const df::hospital_supplies &b) {
-    return a.max_thread == b.max_thread &&
-            a.max_cloth == b.max_cloth &&
-            a.max_splints == b.max_splints &&
-            a.max_crutches == b.max_crutches &&
-            a.max_plaster == b.max_plaster &&
-            a.max_buckets == b.max_buckets &&
-            a.max_soap == b.max_soap;
-}
-
 static const char * get_zone_keys(const df::building_civzonest *zone) {
```

Comment:
This function needs to be rewritten to support the blueprint format for zones and locations. The docs are here: https://docs.dfhack.org/en/latest/docs/guides/quickfort-user-guide.html#zone-mode and here: https://docs.dfhack.org/en/latest/docs/guides/quickfort-user-guide.html#zone-mode-reference